3rdparty/jvm/com/google/guava/BUILD

```diff
@@ -19,7 +19,7 @@ REV = '16.0'
 jar_library(name='guava',
   jars=[
     jar(org='com.google.guava', name='guava', rev=REV,
-        apidocs='http://docs.guava-libraries.googlecode.com/git-history/v%s/javadoc/' % REV),
+        apidocs='https://docs.guava-libraries.googlecode.com/git-history/v%s/javadoc/' % REV),
```

Comment:
Link is broken.

Reply:
Indeed, this the warning I get when going to the site:

<img width="727" alt="Screen Shot 2019-06-06 at 3 31 38 PM" src="https://user-images.githubusercontent.com/14852634/59070600-2f224300-8870-11e9-80da-25b6dfaf4b97.png">

This looks like the right link: https://javadoc.scijava.org/Guava/. I couldn't find the version control for the API, so you would remove the `% REV` piece.